src/flow_id.rs

```diff
@@ -3,8 +3,8 @@ use std::hash::{Hash, Hasher};
 use std::net::{IpAddr, Ipv4Addr};
```

Comment:
I would prefer a git commit --amend into the previous commit, to keep a clear git history :)

Reply:
👍